repository/src/lib.rs

```diff
@@ -415,13 +408,103 @@ impl<T: RawRepository> DistributedRepository<T> {
     ///
     /// Note that the last block will be verified by the given `last_block_proof`,
```

Comment:
fix this please there is no variable named last_block_proof here

Reply:
fixed.

Reply:
did you push? I can't see the difference

Reply:
I force-pushed it now. Sorry.

Reply:
I didn't mean to just delete it. I was suggesting you to update the comment to the current context. 

Fix properly please. There is a left over sentence next line.

Reply:
I changed it. Check current version please.